openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -414,12 +414,19 @@ There are two possible ways to request issuance of a specific Credential type in
 The request parameter `authorization_details` defined in Section 2 of [@!RFC9396] MUST be used to convey the details about the Credentials the Wallet wants to obtain. This specification introduces a new authorization details type `openid_credential` and defines the following parameters to be used with this authorization details type:
 
 * `type`: REQUIRED. String that determines the authorization details type. It MUST be set to `openid_credential` for the purpose of this specification.
-* `credential_configuration_id`: REQUIRED. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
+* `credential_configuration_id`: String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
+* `format`: String identifying the format of the Credential the Wallet needs. This Credential format identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. This specification defines Credential Format Profiles in (#format_profiles).
```

Comment:
```suggestion
* `credential_configuration_id`: OPTIONAL. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles). This parameter MUST NOT be present if `format` parameter is present.
* `format`: OPTIONAL. String identifying the format of the Credential the Wallet needs. This Credential format identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. This specification defines Credential Format Profiles in (#format_profiles). This parameter MUST NOT be present if `credential_configuration_id` parameter is present.
```

Reply:
Would this work for you? Sounds kind of weird to me but I'd definitely prefer some text with "REQUIRED when <..>"
```suggestion
* `credential_configuration_id`: REQUIRED when `format` is not present and MUST NOT be present if `format` parameter is present. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
* `format`: REQUIRED when `credential_configuration_id` is not present and MUST NOT be present if `credential_configuration_id` parameter is present. String identifying the format of the Credential the Wallet needs. This Credential format identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. This specification defines Credential Format Profiles in (#format_profiles).
```

Reply:
we looked at OID4VP and in the similar situations, such as `presentation_definition` and `presentation_definition_uri`, we have not put neither REQUIRED nor OPTIONAL.
https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-5-8.2

Reply:
```suggestion
* `credential_configuration_id`: Either `credential_configuration_id` or `format` MUST be present in an authorization details object of type `openid_credential`. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
* `format`: Either `credential_configuration_id` or `format` MUST be present in an authorization details object of type `openid_credential`. String identifying the format of the Credential the Wallet needs. This Credential format identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. This specification defines Credential Format Profiles in (#format_profiles).
```

Reply:
```suggestion
* `credential_configuration_id`: REQUIRED when `format` parameter is not present. It MUST NOT be present if `format` parameter is present. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
* `format`: REQUIRED when `credential_configuration_id` parameter is not present. It MUST NOT be present if `credential_configuration_id` parameter is present. String identifying the format of the Credential the Wallet needs. This Credential format identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. This specification defines Credential Format Profiles in (#format_profiles).
```